docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,124 @@ across groups and also the difference and ratio between the maximum and minimum:
     >>> print("ratio in recall = ", grouped_metric.ratio(method='between_groups'))    
     ratio in recall =  0.0
 
+
+Fairness metrics
+----------------
+
+Parity constraints
+^^^^^^^^^^^^^^^^^^
+In the sections below, we review the most common fairness metrics, as well
+as their underlying assumptions and suggestions for use. Each metric requires
+that some aspects of the predictor behavior be comparable across groups. In 
+the mathematical definitions below, let :math:`X` denote a feature vector 
+used for predictions, :math:`A` be a single sensitive feature (such as age 
+or race), and :math:`Y` be the true label. Parity metrics are phrased in 
+terms of expectations with respect to the distribution over :math:`(X,A,Y)`.
+
+Demographic parity
+~~~~~~~~~~~~~~~~~~
+Demographic parity is also known as *statistical parity*. A classifier
+:math:`h` satisfies demographic parity under a distribution over
+:math:`(X, A, Y)` if its prediction :math:`h(X)` is statistically
+independent of the sensitive feature :math:`A`. This is equivalent to
+:math:`\E[h(X) \given A=a] = \E[h(X)] \quad \forall a`. :footcite:`agarwal2018reductions`
+In the case of regression, a predictor :math:`f` satisfies demographic parity
+under a distribution over :math:`(X, A, Y)` if :math:`f(X)` is independent
+of the sensitive feature :math:`A`. This is equivalent to
+:math:`\P[f(X) \geq z \given A=a] = \P[f(X) \geq z] \quad \forall a, z`.
+:footcite:`agarwal2019fair`
+
+Demographic parity can be used to assess the extent of allocation harms, as it 
+reflects an assumption that resources should be allocated proportionally 
+across groups. Of the metrics described in this section, it can be the easiest 
+to implement. However, operationalizing fairness using demographic parity 
+rests on assumptions: that either the dataset is not a good representation of 
+what the world actually looks like (e.g., a resume assessment system that is 
+more likely to filter out qualified female applicants due to an organizational 
+bias towards male applicants, regardless of skill level), or that the dataset 
+is an accurate representation of the world, but the world is unjust. 
+
+It's worth considering whether the assumptions underlying this measurement 
+model maintain construct validity (see :ref:`construct_validity`). 
+Consequential validity, for example, asks how the world shaped by using the 
+measurement model; in this case, the relevant question would be whether 
+demographic parity meets the criteria for establishing "fairness", itself 
+an unobservable theoretical construct, and whether satisfying demographic 
+parity brings us closer to the world we'd like to see. 
+
+Equalized odds
+~~~~~~~~~~~~~~
+A classifier :math:`h` satisfies equalized odds under a
```

Comment:
Similar as with previous paragraph, I'd suggest to start with the more intuitive explanation, i.e., we want the model to perform equally well for different groups, then why we would want that (e.g., to avoid allocation / quality of service harms).

Reply:
Done! Please let me know your thoughts.

> The goal of the equalized odds fairness metric is to ensure a machine learning model performs equally well for different groups. It is stricter than demographic parity because it requires that the machine learning model's predictions are not only independent of sensitive group membership, but that groups have the same false positive rates and and true positive 
rates. This distinction is important because a model could achieve demographic parity (i.e., its predictions could be independent of sensitive group membership), but still generate more false positive predictions for one group versus others. Equalized odds does not create the selection issue discussed in the demographic parity section above. For example, in the hiring scenario where the goal is to choose applicants from group A and group B, ensuring the model performs equally well at choosing applicants from group A and B can circumvent the issue of the model optimizing by selecting applicants from one group at random.